sms-hello-world.js

```diff
@@ -8,12 +8,17 @@ const freeclimbSDK = require('@freeclimb/sdk')
 const port = process.env.PORT || 80
 const accountId = process.env.ACCOUNT_ID
 const apiKey = process.env.API_KEY
-const freeclimb = freeclimbSDK(accountId, apiKey)
+const freeclimbConfig = freeclimbSDK.createConfiguration({accountId, apiKey})
+const apiInstance = new freeclimbSDK.DefaultApi(freeclimbConfig);
 
 app.post('/incomingSms', (req, res) => {
-  let to = 'example to number'
-  let from = 'example from number'
-  freeclimb.api.messages.create(from, to, 'Hey! It is your application!').catch(err => {console.log(err)})
+    const { from: userPhoneNumber } = req.body
+    messageRequest = {
```

Comment:
does this need a `const`

Reply:
Not for any reason in particular I suppose!

Reply:
Ah I see, just updated it